packages/babel-helpers/src/helpers/defineAccessor.js

```diff
@@ -0,0 +1,8 @@
+/* @minVersion 7.20.6 */
```

Comment:
Not sure if this is the correct `minVersion`...

Reply:
We will need to update it when releasing, could you update the Makefile like I did in https://github.com/babel/babel/commit/80c6889342160b783f5eb309055685d95e683317?

Reply:
Done in 16758f36e8013998b4d80bce0dfc68ed8aa1df6e.